scripts/run-exawind-nightly-tests.sh

```diff
@@ -41,7 +41,7 @@ printf "\nActivating Spack-Manager...\n"
 cmd "source ${SPACK_MANAGER}/start.sh && spack-start"
 
 printf "\nCleaning Spack installation...\n"
-cmd "sm-clean"
+cmd "spack clean --all"
```

Comment:
This is okay, but if we intend to continue a __clean__ operation as part of the nightlies then there are a few other things in `sm-clean` we may want to preserve like clearing out the python caches inside our repo: https://github.com/psakievich/spack-manager/blob/8f3377ee5e287d8a2c4aa5867541131cfb1fb9cb/start.sh#L203

This has caused issues when updating spack in the past.  The code suggestion is everything in `sm-clean` but the `rm -rf ~/.spack` command.  Overall I think it would be easier to add a flag to `sm-clean` which will cause the command to omit removing `~/.spack`.

```suggestion
  echo "Remove Spack-Manager cache:"
  cmd "rm -rf ${SPACK_MANAGER}/.cache"
  echo "Use the native Spack clean system":
  cmd "spack clean --all"
  echo "Cleaning out pycache from spack-manager repos:"
  cmd "find ${SPACK_MANAGER}/repos -type f -name '*.py[co]' -delete -o -type d -name __pycache__ -delete"
```